site/content/docs/main/restore-hooks.md

```diff
@@ -174,14 +175,15 @@ Below are the annotations that can be added to a pod to specify exec restore hoo
     * The container name where the hook will be executed. Defaults to the first container. Optional.
 * `post.hook.restore.velero.io/command`
     * The command that will be executed in the container. Required.
+    * The command that will be executed in the container. This command is not executed within a shell by default. If a shell is needed to run your command, include a shell command, like `/bin/sh`, that is supported by the container at the beginning of your command. If you need multiple arguments, specify the command as a JSON array, such as `["/usr/bin/uname", "-a"]`. See [Exec Restore Hooks As Pod Annotation Example](#exec-restore-hooks-as-pod-annotation-example). Optional.
```

Comment:
Line 178 conflicts with line 177. Line 178 says the command is optional but 177 says it is required.